webpack/webpack.config.server.js

```diff
@@ -3,7 +3,13 @@ import webpack from 'webpack';
 import HappyPack from 'happypack';
 
 const root = process.cwd();
-const serverInclude = [path.join(root, 'src', 'server'), path.join(root, 'src', 'universal')];
+const serverInclude = [
+  path.join(root, 'src', 'server'),
+  path.join(root, 'src', 'universal')
+];
+const serverExclude = [
```

Comment:
Ah good! I thought that might be the case.

I'll just make sure to keep the piping config (in `server.babel.js`) ignoring it... no sense on reloading the app when a migration file changes.